chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,13 @@
+﻿$ErrorActionPreference = 'Stop';
```

Comment:
I see nothing else in this PR references `chocolateyinstall.ps1`. I guess it's a standard filename that chocolatey knows and uses?